### PR TITLE
fix(asset-cfg-tx): allow params to be optional in that

### PR DESCRIPTION
**Pull request by gskapka**
...per title.

Algorand allows empty params in such a tx so as to signify the destruction of said asset ([See here in go-algorand](https://github.com/algorand/go-algorand/blob/877090be830c2003ff9e44b56ed5a8f355258ae5/data/transactions/asset.go#L33))

### CODE REVIEW

**Review by oliviera9 — DISMISSED** [review]
LGTM

File: src/algorand_transactions/test_utils/mod.rs [hunk 1]
```diff
@@ -125,4 +127,10 @@ mod tests {
     fn should_get_sample_acfg_tx_json_string() {
         get_sample_acfg_tx_json_string();
     }
+
+    #[test]
+    fn should_get_sample_block_with_no_params_in_asset_cfg_tx() {
+        let r = AlgorandBlock::from_str(&read_to_string(&get_path_n(6).unwrap()).unwrap());
+        assert!(r.is_ok())
```
Comment (oliviera9) [inline src/algorand_transactions/test_utils/mod.rs]: Can we add a check for the block hash?

Comment (gskapka) [reply src/algorand_transactions/test_utils/mod.rs]: Good idea - done!

File: src/algorand_transactions/test_utils/mod.rs [hunk 2]
```diff
@@ -125,4 +127,13 @@ mod tests {
...
         get_sample_acfg_tx_json_string();
     }
+
+    #[test]
+    fn should_get_sample_block_with_no_params_in_asset_cfg_tx() {
+        let block = AlgorandBlock::from_str(&read_to_string(&get_path_n(6).unwrap()).unwrap()).unwrap();
+        let hash = block.hash().unwrap();
+        // NOTE: See: https://algoexplorer.io/block/34866583
+        let expected_hash = AlgorandHash::from_base_32("BNAZSMQLCEPRIUTYE44Q6I2XGAAVAOAU3X45VQEYRGIXJVWEKBPQ").unwrap();
+        assert_eq!(hash, expected_hash);
```
Comment (oliviera9) [inline src/algorand_transactions/test_utils/mod.rs:140]: Great!

**Review by oliviera9 — DISMISSED** [review]
LGTM

**Pull request merged**